src/apps/users/services/core.py

```diff
@@ -92,7 +90,7 @@ async def send_password_recovery(
 
         message = MessageSchema(
             recipients=[user.email_encrypted],
-            subject=subject,
+            subject="Password reset",
```

Comment:
Maybe it makes sense to move this string somewhere as constant?

Reply:
Sure. Where do you think would be a good place to put this? I don't see any email related config files. Maybe [service.py](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/M2-5332-password-reset-email-subject/src/config/service.py)?

Reply:
@binho sure. I don't think the email body is really being localized though, unless I'm just not familiar with how it works. Where should I put the translated string?

Reply:
looks like it's `src/locale/en_US` and `src/locale/fr_FR` but this file seems to be automatically generated 🤔

Reply:
That's what it looks like to me as  well. I'll ask about this in Slack